plenum/server/replica.py

```diff
@@ -1122,6 +1122,12 @@ def report_suspicious(reason):
                              "then the latest one - ignoring it".format(key))
         elif why_not == PP_CHECK_REQUEST_NOT_FINALIZED:
             non_fin_reqs = self.nonFinalisedReqs(pre_prepare.reqIdr)
+            for req in non_fin_reqs:
+                if req not in self.requests and self.node.seqNoDB.get(req) != (None, None):
```

Comment:
This should be discuss because it's not obviously for backup replicas.

Reply:
`if req not in self.requests` - this condition can never be True here, since `nonFinalisedReqs` only returns requests which are in `self.requests`